File: backend/BusinessService/Common/Bases/Requests/CmUploadRequestBase.cs
// *******************************************************************
// 業務名称　: 健康管理システム
// 機能概要　: アップロード処理
// 　　　　　　リクエストインターフェースベース
// 作成日　　: 2023.02.14
// 作成者　　: 蔣
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.Service
{
    public class CmUploadRequestBase : DaRequestBase 
    {
        public List<DaFileModel> files { get; set; } = new List<DaFileModel>();     // ファイルリスト
    }
}

File: backend/BusinessService/Common/Bases/Requests/CmUploadRequestBase.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: アップロード処理
' 　　　　　　リクエストインターフェースベース
' 作成日　　: 2024.07.14
' 作成者　　: 蔣
' 変更履歴　:
' *******************************************************************
Namespace JBD.GJS.Service
    Public Class CmUploadRequestBase
        Inherits Db.DaRequestBase
        Public Property files As List(Of Db.DaFileModel) = New List(Of Db.DaFileModel)()     ' ファイルリスト
    End Class
End Namespace
